IntegrationTests/Fixtures/XCBuild/Libraries/Bar/Package.swift

```diff
@@ -5,7 +5,7 @@ import PackageDescription
 let package = Package(
     name: "Bar",
     products: [
-        .library(name: "BarLib", type: .dynamic, targets: ["BarLib"]),
+        .library(name: "BarLibProduct", type: .dynamic, targets: ["BarLib"]),
```

Comment:
Just a nitpick: Would it be valuable to add a `FIXME` comment to each of these changes? Just in case they get lost. And I think it help other code readers understand that some cases are not covered in the test.

Reply:
Thanks, that's a good idea. I think I'll do that in a follow up, just to avoid having CI broken for longer.